src/cli/commands/migrate.ts

```diff
@@ -1,15 +1,25 @@
-import { createConnection } from 'typeorm';
+import { createConnection, LoggerOptions } from 'typeorm';
```

Comment:
suggestion: Would be nice to look into medusa and apply the same concept for the feature flag, so that with the extender migration command you can also run the medusa migration and that the feature flags are taken into account and also usable for custom migrations. wdyt?

Reply:
well in the sequence medusa migrations need to run before medusa extender migrations, the feature flag is automatically handled isn't it ?

Reply:
@adrien2p  please let me know if anything else needs to be done